test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -54,4 +54,9 @@
     <PackageReference Include="Microsoft.ML.TestModels" Version="$(MicrosoftMLTestModelsPackageVersion)" />
     <PackageReference Include="System.Data.SqlClient" Version="$(SystemDataSqlClientVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="column_name_test\model.onnx">
```

Comment:
We shouldn't add large files into the repo. Instead, we use https://github.com/dotnet/machinelearning-testdata, which builds NuGet packages that we consume in this repo.

There are a few ONNX models already in the `Microsoft.ML.Onnx.TestModels` nuget package, that you can reuse in your test. No need to add a new one.